cherrypy/tutorial/tutorial.conf

```diff
@@ -1,4 +1,4 @@
-[global]
-server.socket_host = "127.0.0.1"
-server.socket_port = 8080
-server.thread_pool = 10
+[global]
```

Comment:
Why did you change this file?

Reply:
There was a previous comment from @jaraco about it, before this PR the tutorials used to listen in 127.0.0.1. I reviewed all comments and thought to address this one, as every thing else seems fine.

Reply:
But the diff shows that you changed *every* line in this file. I think you messed up the line endings.

Reply:
Yes, so this change should be reverted.